ast.ts

```diff
@@ -49,5 +52,9 @@ export enum BinOp { Plus, Minus, Mul, IDiv, Mod, Eq, Neq, Lte, Gte, Lt, Gt, Is,
 export enum UniOp { Neg, Not };
 
 export type Value =
-    Literal
-  | { tag: "object", name: string, address: number}
+    //Literal
```

Comment:
Why not still have `Literal`?

Reply:
Resolved in class. We can't have {tag:"string", value:string} in Value since PyValue takes a number parameter.